utils/proxy/_deserializer.py

```diff
@@ -193,13 +192,7 @@ def json_load():
             if content_type_part.startswith("application/json"):
                 item["content"] = json.loads(part.content)
 
-            elif content_type_part == "application/gzip":
-                with gzip.GzipFile(fileobj=io.BytesIO(part.content)) as gz_file:
-                    content = gz_file.read()
```

Comment:
Handle any gzip error here, and if it happen, add property like : 

```
        item["system-tests-error"] = "Can't uncompress gzip data"
```

And exploit this in the test code

Reply:
Done 9238279b74ac91ea573e1cce1c74ea0d5843c394